dicomImport/matRad_importDicomCt.m

```diff
@@ -197,16 +198,24 @@
 ct.dicomInfo.Height                  = ctInfo(1).Height;
 ct.dicomInfo.RescaleSlope            = ctInfo(1).RescaleSlope;
 ct.dicomInfo.RescaleIntercept        = ctInfo(1).RescaleIntercept;
-if isfield(completeDicom,'PatientName')
+if isfield(completeDicom,'PatientName') && dicomBool == true
     ct.dicomInfo.PatientName         = completeDicom.PatientName;
 end
 
+% do not remove the complete dicom information here because ...
+% calcWaterEqD needs some information
 ct.dicomInfo_org = completeDicom;
+
```

Comment:
This is not acceptable. Then you have all identifiers in this fielda and the pseudonymization is useless. We need to make the effort and extract the relevant information and discard everything else. What is actually needed by calcWaterEqD? Should not be too much...